lib/msal-node/src/cache/CacheManager.ts

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Storage } from './Storage';
+import {
+    ClientAuthError, StringDict,
+    StringUtils
+} from '@azure/msal-common';
+import { InMemoryCache, JsonCache } from "cache/serializer/SerializerTypes";
+import { ICachePlugin } from './ICachePlugin';
+import { Deserializer } from "./serializer/Deserializer";
+import { Serializer } from "./serializer/Serializer";
+
+const defaultSerializedCache: JsonCache = {
+    Account: {},
+    IdToken: {},
+    AccessToken: {},
+    RefreshToken: {},
+    AppMetadata: {},
+};
+
+/**
+ * In-memory token cache manager
+ */
+export class CacheManager {
```

Comment:
Haven't we thought of changing the name of this class as we have a `CacheManager` in `msal-common` now?

Reply:
I'm fine with changing the name.

Reply:
Looks like it is changed